### DISCUSSION TITLE
When pressing fogot password, what does content mean

### DISCUSSION THREAD

**Discussion by ahadcove** [WeblateOrg/weblate#5960] [Q&A]
When we go to forgot password there's a field asking for content. What does this mean?

**Comment by nijel** [comment]
That means that you should upgrade to 4.6.1 ;-).

**Reply by ahadcove** [reply]
We already are on 4.6

**Reply by nijel** [reply]
This is fixed in 4.6.1, in 4d197a5493f1cbceda12f4540a42a389621dfb0c.
